ckanext/data_qld/templates/reporting/snippets/admin_report.html

```diff
@@ -50,6 +50,18 @@ <h2>Organisation: {{ org_title }}</h2>
                 count=metrics.overdue_datasets,
                 link='/dashboard/reporting/datasets/' + org_id + '/overdue-datasets?report_type={0}'.format(report_type),
                 report_type=report_type %}
+            {% snippet "reporting/snippets/linked_metric.html",
+                metric="datasets_no_groups",
+                label='Datasets not added to group/s',
+                count=metrics.datasets_no_groups,
+                link='/dashboard/reporting/datasets/' + org_id + '/datasets_no_groups?report_type={0}'.format(report_type),
```

Comment:
Why use string concatenation for org_id and `format` for report_type? Simpler to use `format` for both.

Reply:
It was just copied and pasted from the other reports but I cannot see why it was implemented that way.
Updated to use string format function and set as a variable.